### PR TITLE
senPyDocker

### PR DESCRIPTION
**Pull request by thekeele**
docker_docker_docker
## Adds docker support to make the VCs go wild

- Dockerfile contains the secret sauce to create a docker image

- Images must be built then ran. Running `make prod` will build the image and start a container.

- The result of `make prod` is the same as `make dev` except senPyTweet is running inside a docker container. Query the API as before for testing.

- You will have to install docker. I'm running Docker 18.09.2.

### CODE REVIEW

**Review by rcehemann — APPROVED** [review]
Works like a charm! :ok_hand: 
I had to add scikit-learn to the requirements file.. Not sure how you got it to work on your end unless you forgot to push an updated `requirements.txt`. Will have you ELI5 the dockerfile syntax soon
image

**Comment by thekeele** [comment]
I only tested with `spacey` 😮. I don't think `scikit-learn` ever made it to master. Either way this looks good now.
